src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -439,7 +439,7 @@ Report(
     ReportPrometheusMetrics(summary.metrics.front());
   }
 
-  if (summary.overhead_pct > overhead_pct_threshold) {
+  if (summary.overhead_pct > (double)overhead_pct_threshold) {
     std::cout << "[WARNING] Perf Analyzer is not able to keep up with the "
```

Comment:
Should there be a test added to test_inference_profiler.cc to check that this warning can get triggered?